<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Введите название магазина:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="name"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Введите адрес магазина:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="address"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add_name">
        <property name="text">
         <string>Добавить магазин</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="table_shop">
        <column>
         <property name="text">
          <string>id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>shop</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>address</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_save_data">
        <property name="text">
         <string>Сохранить изменения в базу данных</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
